### PR TITLE
#72 MarkAsPaidController Status

### PR DESCRIPTION
**Pull request by parvinder09**

### CODE REVIEW

**Comment by magwas** [comment]
You have PMD violations. If youo need help, just ask Tamás or me.

File: src/main/java/com/kodekonveyor/completion/MarkAsPaidController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import com.kodekonveyor.work_request.WorkRequestEntity;
+import com.kodekonveyor.work_request.WorkRequestRepository;
+import com.kodekonveyor.work_request.WorkRequestStatusEnum;
+
+@RestController
+public class MarkAsPaidController {
+
+  @Autowired
+  private WorkRequestRepository repository;
```
Comment (szirbucz) [inline src/main/java/com/kodekonveyor/completion/MarkAsPaidController.java]: remove the private access modifier

File: src/test/java/com/kodekonveyor/completion/MarkAsPaidControllerStatusesTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import com.kodekonveyor.work_request.WorkRequestStatusEnum;
+
+@RunWith(MockitoJUnitRunner.class)
```
Comment (szirbucz) [inline src/test/java/com/kodekonveyor/completion/MarkAsPaidControllerStatusesTest.java]: Add the following annotations:
```
@TestedBehaviour("statuses")
@TestedService("MarkAsPaidController")
```

**Comment by magwas** [comment]
You have pmd violations. You can ask help from Tamás.

File: src/main/java/com/kodekonveyor/completion/MarkAsPaidController.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+
+import com.kodekonveyor.work_request.WorkRequestEntity;
+import com.kodekonveyor.work_request.WorkRequestRepository;
+import com.kodekonveyor.work_request.WorkRequestStatusEnum;
+
+@Controller
```
Comment (szirbucz) [inline src/main/java/com/kodekonveyor/completion/MarkAsPaidController.java]: Sorry, I mislead you in my former comment. The annotation that we use is `@RestController`.

**Comment by szirbucz** [comment]
testcases = 2
please fill in https://docs.google.com/forms/d/e/1FAIpQLSdNMbUDPHp2CCbsKtx3qMDCGXIYmmRsjJoF201Xtl-xvb2llQ/viewform

**Pull request merged**